xds/internal/balancer/clusterimpl/clusterimpl.go

```diff
@@ -54,8 +52,11 @@ const (
 )
 
 var (
-	connectedAddress  = internal.ConnectedAddress.(func(balancer.SubConnState) resolver.Address)
-	errBalancerClosed = fmt.Errorf("%s LB policy is closed", Name)
+	connectedAddress = internal.ConnectedAddress.(func(balancer.SubConnState) resolver.Address)
+	// Below function is no-op in actual code, but can be overridden in
+	// tests to give tests visibility into exactly when certain events happen.
+	clientConnUpdateHook = func() {}
+	newPickerUpdated     = func() {}
```

Comment:
For consistency, could you please name this `pickerUpdateHook`.

Reply:
Done